cmd/rook/api.go

```diff
@@ -67,11 +65,17 @@ func startAPI(cmd *cobra.Command, args []string) error {
 		terminateFatal(fmt.Errorf("failed to init k8s client. %+v\n", err))
 	}
 
+	// Using the rook-operator image to deploy other rook pods
+	rookImage, err := k8sutil.GetContainerImage(clientset)
+	if err != nil {
+		terminateFatal(fmt.Errorf("failed to get container image."))
```

Comment:
should you print the error as well?

Reply:
`terminateFatal` will print the message and then exit the process

Reply:
I mean.. Print the error that causes the failure.. `err`.

Reply:
oh right, good catch